sdk/eventhubs/azure-messaging-eventhubs/migration-guide.md

```diff
@@ -247,7 +247,7 @@ public class MyEventProcessor implements IEventProcessor {
     public void onEvents(PartitionContext context, Iterable<EventData> events) {
         System.out.println("Received events from partition: " + context.getPartitionId());
         for (EventData event : events) {
-            System.out.println("Sequence number: " + event.getSystemProperties().getSequenceNumber());
+            System.out.println("Sequence number: " + event.getSequenceNumber());
```

Comment:
Please revert this file. It won't apply after you address this comment: https://github.com/Azure/azure-sdk-for-java/pull/22314/files#r658924971

Reply:
According to your comment, revert this file within new version.